backend/user-service/src/server.ts

```diff
@@ -1,11 +1,38 @@
 import 'dotenv/config'
 import http, { Server } from 'http'
+import { connect } from 'mongoose'
+import Winston, { Logger } from 'winston'
 import index from './index'
 
+// Setup Logger
+export const logger: Logger = Winston.createLogger({
+    level: 'info',
+    format: Winston.format.json(),
+    defaultMeta: { service: 'user-service' },
+    transports: [
+        new Winston.transports.Console(),
+        new Winston.transports.File({ filename: 'error.log', level: 'error' }),
+        new Winston.transports.File({ filename: 'combined.log' }),
+    ],
+})
+
 const port: string = process.env.PORT ?? '3000'
+const dbUrl: string | undefined = process.env.DB_URL
```

Comment:
could you please share in the telegram group the environment variables as well, thanks!!